src/main/java/org/fundacion/jala/converter/models/ProjectSQL.java

```diff
@@ -13,14 +13,18 @@
 import javax.persistence.EntityManagerFactory;
 import javax.persistence.Persistence;
 
+/**
+ * This class is to query to database.
+ */
 public class ProjectSQL {
 
     private static EntityManagerFactory emf = Persistence.createEntityManagerFactory("jalaPersistence");
 
     /**
-     * This method is for to insert dates for database
-     * @param projectName name the archive
-     * @param pathProject path the archive
+     * This method is for to insert dates for database.
+     *
+     * @param projectName name the archive.
```

Comment:
It would be better to use file instead of archive.

Reply:
ok